src/chat/chat_tests.rs

```diff
@@ -2024,7 +2024,6 @@ async fn test_forward_basic() -> Result<()> {
     let msg = alice.recv_msg(&forwarded_msg).await;
```

Comment:
I suggest to compare `rfc724_mid` of this message with the one on Bob's side. I guess that was broken and is fixed now

Reply:
The goal of this PR is to have consistent mid724 in draft and sent state, but I added this check anyways